Regular-Question-Answers/most-visited-sector-in-a-circular-track.cpp

```diff
@@ -0,0 +1,65 @@
+// Author: Murat Biberoğlu
+// Question Link: https://leetcode.com/problems/most-visited-sector-in-a-circular-track/
+
+class Solution {
+   public:
+    vector<int> mostVisited(int n, vector<int>& rounds) {
+        // holds the count of visiting for each sector
+        vector<int> counts(n, 0);
+        // number of rounds
+        int round = rounds.size() - 1;
+
+        // for each round increment one the range [start, finish)
+        // finish is not in the range beacuse we want to prevent counting twice
+        // a sector. but we need to increment specially finish of the last
+        // round.
+        for (int r = 0; r < round; r++) {
+            int start = rounds[r];
+            int end = rounds[r + 1];
+
+            // make sectors 0-indexed
+            start--;
+            end--;
+
+            // if start < end we won't go over starting sector
+            // so increment range [start, end)
+            // in else case, we will go over starting sector
```

Comment:
"else" is an adverb, so you cannot write "else case". Either write "in other case" or "in the else block".